source/game.game.bmx

```diff
@@ -895,7 +895,7 @@ Type TGame Extends TGameBase {_exposeToLua="selected"}
 			Local avgMoney:Int = 0
 			For Local i:Int = 1 To 4
 				If i = playerID Then Continue
-				avgMoney :+ GetPlayerFinance(i).GetMoney()
+				Local playerSum:Long = GetPlayerFinance(i).GetMoney()
```

Comment:
Hier hast Du ja schon erkannt: "GetMoney:Long()", warum ist dann "avgMoney" ueberhaupt ein Int?

Reply:
Da müsste man mal in der Commit-Historie nachschauen. Wird korrigiert.

Reply:
Player#GetMoney() ist aber Int!?

Reply:
Da will ich jetzt aber nicht ran.

Reply:
PS: Ich habe gar nicht erkannt, dass GetMoney schon Long liefert, sondern wollte verhindern, dass beim Aufaddieren von Barem und Lizenzwert ein Überlauf passiert.